packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -73,13 +73,11 @@ export const OutboundAction = {
 export interface OutboundShipmentStateShape {
   draft: OutboundShipment;
   sortBy: SortBy<OutboundShipmentRow>;
-  deletedLines: OutboundShipmentRow[];
```

Comment:
I went with tagging on each line with a boolean. I can see an alternative being to have a `deletedLines`/`createdLines`/`updatedLines` arrays which held the `ids` of the lines, or something like that 🤔